robot_smach_states/src/robot_smach_states/human_interaction/entity_from_operator_pose.py

```diff
@@ -94,15 +90,15 @@ def _get_operator(self):
             if persons:
                 persons = sorted(persons, key=lambda x: x.position.z)
                 person = persons[0]
```

Comment:
Sorting an entire list and then take the first element is an expensive form of finding the element with the minimum value.
A cheaper solution is `min(persons, key=lambda p: p.position.z)`
For more details, see https://docs.python.org/2.7/library/functions.html?highlight=min#min